Modify Rule S2757: LaYC =+ (#2188)
SonarSource/rspec: rules/S2757/common/code-example.adoc

[source,java]
----
int target = -5;
int num = 3;

> target =- num;  // Noncompliant: target = -3. Is that really what's meant?

Reviewer: ```suggestion
target =- num;  // Noncompliant: target = -3. Is it the wanted behavior?
```
(and same comment for the other code examples)
